### ISSUE TITLE
Proguard task blocks the build process

### ISSUE THREAD

**Issue by fdasoghe** [jvoegele/gradle-android-plugin#4]
Hello again.

If I set the proguard task to be bypassed (with proguard.enabled = false), I got this error:
- What went wrong:
  Execution failed for task ':androidPackageDebug'.
  Cause: /home/fabio/Android/workspace_3.6.1/clowai/build/libs does not exist.

Reading the plugin's code, it seems that the Proguard task is necessary to create the output jar, which is used by the androidPackageDebug.

If this is the case, maybe it should be refactored in order to have the jar created before the Proguard task, then having optionally Proguard using this jar instead of the project's classes dir.

**Comment by jvoegele** [comment]
Hello think01,

Thanks for the issue reports.  I will look into them as soon as I can, and with luck get a 1.0 release out the door very soon.

**Comment by fdasoghe** [comment]
This problem is nastier then I saw in the first time.

If you do a first build with proguard enabled, it correctly create the jar. Then, if you turn off proguard, the build works fine.

What happens is that gradle find the old jar file and goes on, with the "little" problem that it uses the old compiled and packaged classes!!

**Comment by jvoegele** [comment]
Yes, this is a pretty serious bug. I'm working on a fix.

**Comment by jvoegele** [comment]
I've pushed a fix for this issue to the proguard-work branch.  If you build from that branch you should no longer experience this problem.  Please let me know what your testing reveals.

I will close this issue when you have verified that the fix works.

**Comment by fdasoghe** [comment]
Ok, I'll do as soon as I can.

In the meantime, if anyone is interested, I think I've found a little workaround (for gradle-android-plugin 0.9.3).

In your gradle.build file, put this code:

```
proguard.enabled = false
jar {
    archiveName = proguard.outJar.name
}
androidPackageDebug.dependsOn ':jar'
```

It creates the file that the proguard task doesn't create if disabled, and the rest of the lifecycle can proceed.

Hope that helps.

**Comment by fdasoghe** [comment]
I've checked the plugin on the proguard-work branch and I can confirm it works now with proguard.enabled = false.

Great work! :o)

**Comment by jvoegele** [comment]
Thanks.

For now, this fix is only in the proguard-work branch.  Once I've got the other issues fixed, I will collect this and all of the other outstanding fixes into a 1.0 release.
